scripts/README.md

```diff
@@ -15,28 +15,44 @@ Contains all the scripts relevant to benchmarking `cardano-node`. See the benchm
 #### buildkite
 Contains scripts relevant to IOHK's CI.
 
-#### byron-shelley-allegra-mary
-Contains a script that sets up a cluster beginning in the Byron era and can transition to the Shelley era. You can also start a cluster in the Shelley, Allegra or Mary era by supplying an argument to `mk-files.sh`.
+### Clusters
+
+Most clusters sub-folders contain a `mkfiles.sh` script which creates a top level
```

Comment:
The language here is too indefinite i.e "most", "possibly", "somtimes". I would just start it with "Generally" and then add more details to the specific sections where this doesn't apply.

Reply:
Yes, makes sense :+1: